src/index.html

```diff
@@ -11,12 +11,47 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="./index.html"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul>
+          <li>
+            <a
+              href="#apple"
+              class="is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li><a href="#samsung">SAMSUNG</a></li>
```

Comment:
Fix code styles and add classes everywhere
```suggestion
          <li class="nav__item">
            <a href="#samsung" class="nav__link">SAMSUNG</a>
          </li>
```